src/components/Score.js

```diff
@@ -0,0 +1,18 @@
+class Score {
+    constructor() {
+        this.score = 0;
+    }
+
+    add(increment) {
+        console.assert(increment >= 0, this.Score + increment !== Infinity);
```

Comment:
Why is it wrong to reduce the score?

Reply:
it isn't wrong, it is just not needed. If a person has repaired something how can that type of count be reversed? If we change the score metrics to something else then definitely and we should write a decrement function or modify this to do both.